presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1132,3 +1132,39 @@ In the following query, the expression CURRENT_TIMESTAMP returns the current tim
             20 | canada        |         2 | comment
             30 | mexico        |         3 | comment
     (3 rows)
+
+Sorted Tables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector supports to create sorted files.
+Data in the Iceberg table is sorted during the writing process within each file.
+
+Sorted Iceberg tables can provide a huge increase in performance in query times.
+Sorting is particularly beneficial when the sorted columns show a
+high cardinality and are used as a filter for selective reads.
+
+The sort order is configured with the ``sorted_by`` table property.
+When creating the table, can specify an array of one or more columns to use for sorting.
+The following example creates the table with sorted_by property and sorts file based on field ``join_date``.
+
```

Comment:
A local build shows the first code example is not correctly formatted. Compare the first code block with the second correctly generated code block in the screenshot. 

Would you please reformat the first code block so that it displays in your local build like the second code block? Thanks! 
<img width="964" alt="Screenshot 2024-02-21 at 10 26 24 AM" src="https://github.com/prestodb/presto/assets/7013443/f40bb441-8b06-443b-8295-ab3bf6c8c88c">

Reply:
@steveburnett, I have made the suggested changes.

Reply:
Thanks @evanvdia! I see that you did make changes, including in the first code block here, but a local build of the docs still renders the first code block as though it has formatting problems. Did you not experience this formatting problem when you did a local build of the docs after the change? 
Screenshot from a new pull of the branch and a new build of the docs here just now. 
<img width="962" alt="Screenshot 2024-02-22 at 9 51 52 AM" src="https://github.com/prestodb/presto/assets/7013443/2e38fe22-1a03-4117-8890-bee0d8f60190">

Reply:
Thanks for the formatting change! The first code block looks better now in a local build after your changes.